mne/gui/_coreg_gui.py

```diff
@@ -1265,13 +1298,16 @@ class ViewOptionsPanel(HasTraits):
     mri_obj = Instance(SurfaceObject)
     hsp_obj = Instance(PointObject)
     eeg_obj = Instance(PointObject)
+    hpi_obj = Instance(PointObject)
     view = View(VGroup(Item('mri_obj', style='custom',
-                            label="MRI head"),
+                            label="MRI"),
                        Item('hsp_obj', style='custom',
-                            label="Head shape"),
+                            label="Extra"),
```

Comment:
I don't find "Extra" intuitive – we've been calling them head shape points because they come from the head shape recording, is that misleading in some way?

Reply:
Some places we call these "head shape", in others we call them "extra" because they are extra points during digitization. At some point we should standardize. I was moving toward standardizing around the Elekta standard but maybe we should stick with "head shape".

Reply:
I can def. see how 'extra' is not intuitive and in this context head shape is clearly more so. There is the common case however of EEG electrodes being the only 'head shape' information we have; here whatever points may be acquired (on an Elekta system at least) are likely to be on the face, and consequently more 'extra'-like. +1 for keeping 'head shape' from me though, it's the most natural in this context.